org.eclipse.m2e.tests/src/org/eclipse/m2e/tests/launch/LaunchingUtilsTest.java

```diff
@@ -0,0 +1,71 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Sonatype, Inc.
```

Comment:
Copyright is yours (or your employers, if you contributed during work time), not Sonatype